config/default/editor.editor.minimal_html_with_styles.yml

```diff
@@ -0,0 +1,111 @@
+uuid: 4985794b-697f-48e3-8b25-1cdcb4c4b7ef
+langcode: en
+status: true
+dependencies:
+  config:
+    - filter.format.minimal_html_with_styles
+  module:
+    - ckeditor5
+format: minimal_html_with_styles
```

Comment:
Why not just expand the minimal html instead of extra overhead?

Reply:
This new format is only to be used by the postcard components, while the existing minimal html format is to remain the same for all other components.

Reply:
Sounds like an annoying extra maintenance burden.